apps/atomy-q/API/tests/Unit/Database/Factories/UserFactoryTest.php

```
@@ -0,0 +1,32 @@
+    {
+        $user = User::factory()->active()->create();
+
+        $this->assertSame('active', $user->status);
+    }
+
+    public function test_user_locked_state(): void
+    {
+        $user = User::factory()->locked()->create();
+
+        $this->assertSame('locked', $user->status);
+    }
```

PHPMD (2.15.0)

[error] The method test_user_locked_state is not named in camelCase. (undefined)

(CamelCaseMethodName)